config/src/config/api_config.rs

```diff
@@ -42,4 +42,8 @@ impl ApiConfig {
     pub fn randomize_ports(&mut self) {
         self.address.set_port(utils::get_available_port());
     }
+
+    pub fn content_length_limit(&self) -> u64 {
```

Comment:
will add a field content_length_limit to api config later in a small PR to ensure compatible test pass.